content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -536,7 +547,7 @@ const product = prisma.product.update({
     },
   },
 })
-``` -->
+```
```

Comment:
Missing deleteMany example?

Reply:
I'm seeing an example starting line 533, 'The following example uses `deleteMany` to delete all photos with a `height` of 100:'?